resources/acsl/booleanalgebra.md

```diff
@@ -74,61 +108,132 @@ You can regroup terms in an expression without changing the value of the express
 
 | Identity | Description |
 | --- | --- |
-| x + x = x | If `x = 0`, then `0 + 0 = 0`. If `x = 1`, then `1 + 1 = 1`. Both of these equations are true; thus, the law must be valid. |
-| x • x = x | If `x = 0`, then `0 • 0 = 0`. If `x = 1`, then `1 • 1 = 1`. Again, these equations are both true, thus proving the law. |
+| `x + x = x` | If `x = 0`, then `0 + 0 = 0`. If `x = 1`, then `1 + 1 = 1`. Both of these equations are true; thus, the law must be valid. |
+| `x • x = x` | If `x = 0`, then `0 • 0 = 0`. If `x = 1`, then `1 • 1 = 1`. Again, these equations are both true, thus proving the law. |
 
 ## Annihilator Law
```

Comment:
Just an FYI: Probably add a note that they should intuitively remember these laws (don't try to memorize them, they should just make sense).